### PR TITLE
Add 7.15mm EP for QFN-64

### PR DESCRIPTION
**Pull request by matthuszagh**

### CODE REVIEW

File: scripts/Packages/Package_DFN_QFN/size_definitions/qfn-64_9x9.yaml [hunk 1]
```diff
@@ -151,6 +151,59 @@ QFN-64-1EP_9x9mm_P0.5mm_EP6x6mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+QFN-64-1EP_9x9mm_P0.5mm_EP7.15x7.15mm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.analog.com/media/en/technical-documentation/data-sheets/229321fa.pdf (page 27)'
```
Comment (poeschlr) [inline scripts/Packages/Package_DFN_QFN/size_definitions/qfn-64_9x9.yaml]: please use the #page=27 feature for the url to point the webbrowser to the correct page.

File: scripts/Packages/Package_DFN_QFN/size_definitions/qfn-64_9x9.yaml [hunk 2]
```diff
@@ -151,6 +151,59 @@ QFN-64-1EP_9x9mm_P0.5mm_EP6x6mm:
...
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    nominal: 9.0
+    tolerance: 0.1
+  body_size_y:
+    nominal: 9.0
+    tolerance: 0.1
```
Comment (poeschlr) [inline scripts/Packages/Package_DFN_QFN/size_definitions/qfn-64_9x9.yaml]: please add the body height

```yaml
body_height:
  nominal: 0.75
  tolerance: 0.05
```

**Comment by matthuszagh** [comment]
Changes made.

**Comment by poeschlr** [comment]
Force pushes make reviewing of changes harder. (I can not click the button for "view changes" above my review.)
So i would suggest to refrain from using them.

**Comment by poeschlr** [comment]
@pointhi the footprint has been merged, this is good to go.

**Pull request merged**